### ISSUE TITLE
"Computer" zim is either too big or too small

### ISSUE THREAD

**Issue by Popolechien** [openzim/mwoffliner#273]
See attached image: the computer zim file is 16Gb large but covers a lot more than Computer science (even though the landing page *is* the computer science article). The zim currently also covers physics, but only part of chemistry and Earth science.

![capture d ecran 2018-04-03 a 15 12 33](https://user-images.githubusercontent.com/21675885/38251361-8a406f3a-3751-11e8-8dd5-395e92d0ca5c.png)

**Comment by kelson42** [comment]
@Popolechien Yes, I'm happy about an alternative selection/welcome page proposal.

**Comment by Popolechien** [comment]
I think the article as welcome page is actually pretty good as is, but what's the selection criteria that the file is both so big and so much beyond simple Computer Science (yet misses some of the sciences)?
